test/end-to-end/vouchers/complex.spec.js

```diff
@@ -6,30 +6,42 @@ const ComplexVoucherPage = require('./complex.page');
 const FU = require('../shared/FormUtils');
 const GU = require('../shared/GridUtils');
 
-describe('Complex Vouchers', function () {
+describe('Complex Vouchers', () => {
   before(() => helpers.navigate('vouchers/complex'));
 
-  it('creates a complex voucher', () => {
+  it('creates a complex voucher', function () {
```

Comment:
I'm just curious to know why you replaced the `() => ` by a `function ()`, is there any reason ?

Reply:
Yeah - arrow functions don't have a `this` object inside them, and on line 16 I use `this.timeout()`.

[According to stack overflow](https://stackoverflow.com/a/15982893), you are supposed to be able to use `it('something', () => { /* ... */ }).timout(45000);`.   That version didn't work for me though... I think our mocha version might not be up to date.

For more information on setting the timeout inside a test, [you can check out Mocha's documentation on it](http://mochajs.org/#test-level).